docs/SemanticARC.md

```diff
@@ -376,12 +434,19 @@ generalized dominators that can be used. If computation of generalized
 dominators is too expensive for normal use, they could be used on specific
 verification bots and used when triaging bugs.
 
+[EE]
+When looking at semantic ARC in terms of local RCs (see my comment above), the verifier is very simple. It's a dataflow algorithm, which ensures that the local RCs are consistent at block entries for all RC roots. It's similar to what we do to check proper nesting of alloc_stacks.
+
+Beside that, I didn't understand what algorithm you propose (what is "disproving joint dominance")?
```

Comment:
We need to prove that along any path through the program, the copy is destroyed only once. That implies proving that the consumers of the value do not have any paths in between them and in addition joint post-dominate the retain set and vis-a-versa.